### ISSUE TITLE
Passive rigidbody is not effected by parent movement

### ISSUE THREAD

**Issue by Simonrazer** [armory3d/armory#1468]
If there is a passive rigidbody which is child of another object and the parent object is being moved via script the passive rigidbody will not follow, even though it should. Because passive rigidbodies can be moved on their own via script I think this is a bug.
[passive rigidbody.zip](https://github.com/armory3d/armory/files/3827457/passive.rigidbody.zip)

**System**
Blender: 2.80
Armory: 2019.10.0 (Blender updated)
OS: Windows 10
Graphics card: GTX 1060 6gb

**Test File**
press h to move the parent of the rigidbody (the floor&cube) and it will not follow. But if you use h to move it on its own it will.

**Comment by QuantumCoderQC** [comment]
Hey,

Please check if you have enabled the `Animated` checkbox for the Passive Rigid-body. Then try moving the parent through a script.

![Capture](https://user-images.githubusercontent.com/55564981/68533330-7cc34000-0327-11ea-8a8d-cb16ccd97896.PNG)

**Comment by Simonrazer** [comment]
That button makes it move but not collide with anything. I guess it should be used when you use rigidbodies as a kind of trigger volume.

**Comment by QuantumCoderQC** [comment]
I believe you must call `object.getTrait(RigidBody).syncTransform();` on the Passive rigid body once the transformation on the parent object is done. 

Please try it.

**Comment by Simonrazer** [comment]
Yeah putting that after the transformation (I used a custom node after "Set Location") did it!
But it did it differently than if the transformation was only done on the rigidbody. When moving the parent upwards and then syncing the floor the floor drags the car sitting on top of it along with it . Nice.
But when transforming it alone it it just phases through the car and the car hovers mid air, I think because physics are not being activated for it. 
So eh I think there is another bug, since these two actions should result in the same situation for the car standing on the floor but they do not.

**Comment by QuantumCoderQC** [comment]
I think we need to call `syncTransform()` even if we move the Passive rigid body directly. I think this is because when we do a `transform()` it is applied to the `object`.

It is either this or that the rate of movement is too fast for the Physics Engine to calculate.
